data_analysis/model_ad/notebooks/MG-40_Create_harmonized_pathology_source_file.rmd

```diff
@@ -129,27 +127,77 @@ download_file <- function(synId, df_name, type) {
 
   cat(paste0("Downloading ", synId, "..."))
 
-
   file <- synGet(id_value, version = version_value, downloadLocation = input_dir, ifcollision='overwrite.local')
   path <- file$path
 
   cat("\nDownload on:  ", date())
   cat("\n", synId, "\n- Modified on ", file$properties$modifiedOn, "\n- Version ", file$properties$versionNumber)
   cat("\npath: ", file$path, "\n")
 
-  if (type == 'f') {
-    data <- read_feather(path)
-  } else {
-    if (type == 'json') { data <- fromJSON(path) }
-    if (type == 'csv' || type == 'table') { data <- read.csv(path) }
-    if (type == 'tsv') { data <- read.csv(path, sep = "\t") }
-    if (type == 'rda') { data <- load(path)}
-    if (type == 'rds') {data <- readRDS(path)}
```

Comment:
Just to check, we are removing all of this because there are only CSV files anyway?

Reply:
Nm, answered my own question.